importer/helpers/helpers.go

```diff
@@ -28,10 +28,11 @@ var roughLinkSize = 34 + 8 + 5   // sha256 multihash + size + no name + protobuf
 // For now, we use:
 //
 //   var roughLinkBlockSize = 1 << 13 // 8KB
-//   var roughLinkSize = 288          // sha256 + framing + name
+//   var roughLinkSize = 34 + 8 + 5   // sha256 multihash + size + no name
+//                                    // + protobuf framing
 //   var DefaultLinksPerBlock = (roughLinkBlockSize / roughLinkSize)
-//
-// See calc_test.go
```

Comment:
Any reason to remove this reference? (also, I'm not even sure where it's pointing to)

Reply:
Sorry, I miss your commit. calc_test.go is deleted in https://github.com/ipfs/go-ipfs/commit/bc79ae17a1987c19e9cd258c4c12d4ba9c3470c6 . calc_test could calculate rough link size,  data block sizes and link block sizes.

Reply:
Wow, this file was removed in 2015.. thanks!